### PR TITLE
fab changes for hqcelery0

### PR DESCRIPTION
**Pull request by snopoke**
@dannyroberts

buddy @sravfeyn

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 12 errors:

* fab/fabfile.py, line 1079 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1080 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1081 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1082 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1083 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1084 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1085 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1086 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1087 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1088 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1089 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1090 - E123 closing bracket does not match indentation of opening bracket's line

File: fab/environments.yml [hunk 1]
```diff
@@ -12,70 +12,60 @@ base:
...
+      reminder_queue:
+        concurrency: 8
+      reminder_rule_queue:
+        concurrency: 4
+      saved_exports_queue:
+        concurrency: 3
+      sms_queue:
+        concurrency: 8
+      flower: {}
+      lucene: {}
```
Comment (snopoke) [inline fab/environments.yml]: India is the only place lucene will be run, is that correct?

**Comment by dimagimon** [comment]
There are 12 errors:

* fab/fabfile.py, line 1079 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1080 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1081 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1082 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1083 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1084 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1085 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1086 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1087 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1088 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1089 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1090 - E123 closing bracket does not match indentation of opening bracket's line

File: fab/environments.yml (hunk 1 above)
Comment (dannyroberts) [inline fab/environments.yml]: I would have thought we don't use lucene anymore period.

Comment (snopoke) [inline fab/environments.yml]: Great, I'll kill it then

File: fab/environments.yml [hunk 2]
```diff
@@ -87,90 +77,75 @@ production:
...
-    reminder_queue:
-      concurrency: 8
-    reminder_rule_queue:
-      concurrency: 4
-    saved_exports_queue:
-      concurrency: 3
-    sms_queue:
-      concurrency: 8
+  celery_processes:
+    hqcelery0:
```
Comment (dannyroberts) [inline fab/environments.yml]: Aha, I see. This is the real change; for the other environments you're just adding `"*"` and indenting.

File: fab/environments.yml [hunk 3]
```diff
@@ -87,90 +77,75 @@ production:
...
+    '*':
+      main:
+        concurrency: 4
+      background_queue:
+        concurrency: 2
+      pillow_retry_queue:
+        concurrency: 1
+      saved_exports_queue:
+        concurrency: 3
+      flower: {}
```
Comment (dannyroberts) [inline fab/environments.yml]: Does just not having queues show up in this list replace e.g. `sms_queue_enabled`?

File: fab/environments.yml (hunk 1 above)
Comment (czue) [inline fab/environments.yml]: :+1: to kill

Comment (snopoke) [inline fab/environments.yml]: https://github.com/dimagi/commcare-hq/pull/6583

**Comment by sravfeyn** [comment]
I just had a good code-reading, instead of reviewing. Not too familiar with these changes, @dannyroberts  please merge this when your comments are resolved.

**Comment by dimagimon** [comment]
There are 11 errors:

* fab/fabfile.py, line 1079 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1080 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1081 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1082 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1083 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1084 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1085 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1086 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1087 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1088 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1089 - E123 closing bracket does not match indentation of opening bracket's line

**Comment by dannyroberts** [comment]
@snopoke assume you'll ping me when this is ready for final review and merge

**Comment by snopoke** [comment]
yup

**Comment by dimagimon** [comment]
There are 11 errors:

* fab/fabfile.py, line 1087 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1088 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1089 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1090 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1091 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1092 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1093 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1094 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1095 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1096 - E241 multiple spaces after ':'
* fab/fabfile.py, line 1097 - E123 closing bracket does not match indentation of opening bracket's line

File: fab/environments.yml (hunk 2 above)
Comment (snopoke) [inline fab/environments.yml]: yup

File: fab/environments.yml (hunk 3 above)
Comment (snopoke) [inline fab/environments.yml]: yes

**Comment by snopoke** [comment]
@dannyroberts I think this is rtg

**Pull request merged**